docs/docs/data-stores/managing-data-stores/README.md

```diff
@@ -0,0 +1,34 @@
+# Managing Data Stores
+
+You can view the contents of your Data Stores at any time in the [Pipedream Dashboard](https://pipedream.com/data-stores/).
+
+On the left had navigation menu, there's a dedicated **Data Stores** section.
+
+From here you can open individual Data Stores by clicking on them individually.
+
+## Editing Data Store values manually
+
+You can also edit Data Store values manually from this view. To edit a Data Stores values, click the pencil icon on the far right of the data store.
```

Comment:
You capitalize "Data Store" here but not in the other example docs. It seems like we're capitalizing in the product, so maybe we should throughout the docs? A lot of style guides I've seen do this — in general they capitalize things that refer to individual, specific products that are specific to their platform. So "Data Store" makes sense to me, but maybe take a look at other style guides and see what they do.